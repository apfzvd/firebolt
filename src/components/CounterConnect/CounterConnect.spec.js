import { shallow } from 'enzyme'
import React from 'react'
import { Counter } from './index'

const wrapper = shallow(<Counter />)

it('snapshot counter connect', () => {
  expect(wrapper.debug()).toMatchSnapshot()
})
