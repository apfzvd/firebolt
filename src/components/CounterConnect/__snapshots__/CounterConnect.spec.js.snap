// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot counter connect 1`] = `
"<div>
  <h1>
    Componente conectado
  </h1>
  <div />
  <Button onClick={[undefined]} disabled={false} color=\\"primary\\" size=\\"medium\\" type=\\"button\\" className={[undefined]} active={false}>
    Aumentar valor
  </Button>
  <Button onClick={[undefined]} disabled={false} color=\\"primary\\" size=\\"medium\\" type=\\"button\\" className={[undefined]} active={false}>
    Diminuir valor
  </Button>
  <Button onClick={[undefined]} disabled={false} color=\\"primary\\" size=\\"medium\\" type=\\"button\\" className={[undefined]} active={false}>
    Zerar
  </Button>
</div>"
`;
